LayoutTests/TestExpectations

```diff
@@ -4906,7 +4906,7 @@ webkit.org/b/220325 http/wpt/css/css-highlight-api/highlight-text-cascade.html [
 imported/w3c/web-platform-tests/css/css-highlight-api/painting/custom-highlight-painting-008.html [ Skip ]
 imported/w3c/web-platform-tests/css/css-highlight-api/painting/css-target-text-decoration-001.html [ ImageOnlyFailure ]
 imported/w3c/web-platform-tests/css/css-highlight-api/painting/custom-highlight-painting-002.html [ ImageOnlyFailure ]
-imported/w3c/web-platform-tests/css/css-highlight-api/painting/custom-highlight-painting-004-2.html [ ImageOnlyFailure ]
+imported/w3c/web-platform-tests/css/css-highlight-api/painting/custom-highlight-painting-004-2.html [ Pass ]
```

Comment:
Why are marking these tests as Pass instead of just removing them from TestExpectations?

We only marked tests as Pass in TestExpectations when they are skipped globally and passing only on a single platform.

Reply:
I did not know that we didn't have to mark them as pass and could just remove. Will do that now!